cypress/integration/common/i-can-enter-owner-information.spec.ts

```diff
@@ -7,7 +7,6 @@ import {
 } from "./selectors-and-assertions.spec";
 
 export const givenIHaveEnteredMyPersonalDetails = (): void => {
-  givenIAmAt(PageURLs.aboutBeaconOwner);
```

Comment:
`givenIAmAt(...)` was sending the browser to the `/register-a-beacon/...` page, discarding the `?useIndex=` parameter.  Something the user on the happy path wouldn't do!  So is removed from all assertions.